test/PreAMMBatcher.test.ts

```diff
@@ -107,6 +108,101 @@ describe("PreAMMBatcher: Unit Tests", () => {
   });
 
   describe("orderChecks()", () => {
+    describe("order validity period", () => {
+      it("succeeds if order is within validity period", async () => {
+        const testCaseInput = baseTestInput(
+          token0,
+          token1,
+          [traderWallet1],
+          [traderWallet2],
+        );
+        const now = (await waffle.provider.getBlock("latest")).timestamp;
+        testCaseInput.sellOrdersToken0[0].validFrom = BigNumber.from(now - 60);
+        testCaseInput.sellOrdersToken0[0].validUntil = BigNumber.from(now + 60);
+        testCaseInput.sellOrdersToken1[0].validFrom = BigNumber.from(now - 60);
+        testCaseInput.sellOrdersToken1[0].validUntil = BigNumber.from(now + 60);
+
+        await expect(
+          batchTester.orderChecksTest(
+            [testCaseInput.sellOrdersToken0[0].getSmartContractOrder()],
+            [testCaseInput.sellOrdersToken1[0].getSmartContractOrder()],
+            { gasLimit: 6000000 },
+          ),
+        ).to.not.be.reverted;
+      });
+      it("fails if sell order for token 0 is expired", async () => {
```

Comment:
nit: I find it hard to read these large blocks of tests with no line breaks 
```suggestion

      it("fails if sell order for token 0 is expired", async () => {
```